infrastructure/src/main/java/org/corfudb/infrastructure/BatchWriter.java

```diff
@@ -43,7 +48,8 @@
      *
      * @param streamLog stream log for writes (can be in memory or file)
      */
-    public BatchWriter(StreamLog streamLog) {
+    public BatchWriter(StreamLog streamLog, boolean doSync) {
```

Comment:
Can you add comments for the new parameter.

Reply:
done.